lib/studies/reply/adaptive_nav.dart

```diff
@@ -312,7 +314,12 @@ class _NavigationRailHeader extends StatelessWidget {
             start: 12,
             end: 16,
           ),
-          child: _ReplyFab(extended: extended),
+          child: Consumer<EmailStore>(
```

Comment:
If _ReplyFab is used more than twice, I'd recommend making the encapsulating Consumer its own widget

Reply:
I'll make this in a separate PR, since this Consumer is used in a lot of different places.

Reply:
To be clear I meant this `Consumer` + `_ReplyFab` subtree, not `Consumer` in general

Reply:
Thank you for the clarification!